docs/docs/guides/getting_started.md

```diff
@@ -182,9 +142,9 @@ If you don't have separate streams for detect and record, you would just add the
 
 By default, Frigate will retain video of all events for 10 days. The full set of options for recording can be found [here](../configuration/index.md#full-configuration-reference).
 
-### Step 8: Enable snapshots (optional)
+### Step 7: Enable snapshots (optional)
```

Comment:
Given that the events guide was removed, I think it would be important to mention here that one of recordings OR snapshots must be enabled for events to snow up